apps/base-docs/docs/tools/oracles.md

```diff
@@ -119,14 +119,39 @@ See [this guide](https://docs.ora.io/doc/oao-onchain-ai-oracle/develop-guide/tut
 
 ## Pyth
 
-[Pyth](http://pyth.network/) offers 250+ [price feeds](https://pyth.network/price-feeds) for Base.
+The [Pyth Network](https://pyth.network/) is one of the largest first-party Oracle network, delivering real-time data across [a vast number of chains](https://docs.pyth.network/price-feeds/contract-addresses). Pyth introduces an innovative low-latency [pull oracle design](https://docs.pyth.network/documentation/pythnet-price-feeds/on-demand), where users can pull price updates onchain when needed, enabling everyone in the onchain environment to access that data point most efficiently. Pyth network updates the prices every **400ms**, making Pyth one of the fastest onchain oracles.
 
-See [this guide](https://docs.pyth.network/documentation/pythnet-price-feeds/evm) to learn how to use the Pyth feeds.
+#### Pyth Price Feeds Features:
 
-#### Supported Networks
+- 400ms latency
+- Efficient and cost-effective Oracle
+- [First-party](https://pyth.network/publishers) data sourced directly from financial institutions
+- [Price feeds ranging from Crypto, Stock, FX, Metals](https://pyth.network/developers/price-feed-ids)
+- [Available on all major chains](https://docs.pyth.network/price-feeds/contract-addresses)
 
-- Base Mainnet
-- Base Sepolia (Testnet)
+#### Supported Networks for Base (Pyth Price Feeds):
+- Base Mainnet: [`0x8250f4aF4B972684F7b336503E2D6dFeDeB1487a`](https://basescan.org/address/0x8250f4aF4B972684F7b336503E2D6dFeDeB1487a) 
+- Base Sepolia: [`0xA2aa501b19aff244D90cc15a4Cf739D2725B5729`](https://base-sepolia.blockscout.com/address/0xA2aa501b19aff244D90cc15a4Cf739D2725B5729)
+
+
+### Pyth Entropy
+Pyth Entropy allows developers to quickly and easily generate secure **random numbers** onchain.
+
+Check [how to generate random numbers in EVM contracts]((https://docs.pyth.network/entropy/generate-random-numbers/evm)) for a detailed walkthrough.
```

Comment:
@aditya520 could you remove the duplicate parentheses around the link here? this will cause the link to break.